packages/client/hmi-client/src/types/Types.ts

```diff
@@ -88,6 +88,15 @@ export interface Artifact extends TerariumAsset {
     metadata?: any;
 }
 
+export interface ChartAnnotation extends TerariumAsset {
+    nodeId: string;
+    outputId: string;
```

Comment:
What is a chart annotation outputId used for?

Reply:
I think it is meant to map to the outputport-id